From 7167a7364150eeffb7c125f6df82d4e5d4328bb8 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Sun, 26 Jul 2020 19:16:21 +0200
Subject: [PATCH 18/30] firewall: tune lan/wan into named sections

We want to reference these from within OpenSync. They need to be named for this to work.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/network/config/firewall/files/firewall.config | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/network/config/firewall/files/firewall.config b/package/network/config/firewall/files/firewall.config
index 8874e9882c..58ca162cc7 100644
--- a/package/network/config/firewall/files/firewall.config
+++ b/package/network/config/firewall/files/firewall.config
@@ -6,14 +6,14 @@ config defaults
 # Uncomment this line to disable ipv6 rules
 #	option disable_ipv6	1
 
-config zone
+config zone lan
 	option name		lan
 	list   network		'lan'
 	option input		ACCEPT
 	option output		ACCEPT
 	option forward		ACCEPT
 
-config zone
+config zone wan
 	option name		wan
 	list   network		'wan'
 	list   network		'wan6'
@@ -23,7 +23,7 @@ config zone
 	option masq		1
 	option mtu_fix		1
 
-config forwarding
+config forwarding lan_fw
 	option src		lan
 	option dest		wan
 
-- 
2.25.1

